### PR TITLE
Update Route module

### PR DESCRIPTION
**Pull request by phamtuan2993**
+ toRoute and toUrl to not add redundant /#/ into url

http://localhost:1234/#/books => http://localhost:1234/books

`<Link route=Config.Home>` does not work => Work now

### CODE REVIEW

**Comment by phthhieu** [comment]
@phamtuan2993 I dont think we would update this way, using hash location will help us release the website into a serving static file server such as s3, now.sh

File: client/src/Route.re [hunk 1]
```diff
@@ -31,7 +30,7 @@ module Config = {
 
   let toUrl =
     fun
-    | Home => "#/"
```
Comment (phamtuan2993) [inline client/src/Route.re]: This is redundant and is what makes Config.Home does not work

File: client/src/Route.re [hunk 2]
```diff
@@ -19,7 +19,6 @@ module Config = {
     | []
     | ["/"] =>
       switch (hashes) {
-      | []
```
Comment (phamtuan2993) [inline client/src/Route.re]: This is redundant case, `hashes` will always have at least 1 member of empty string due to `url.hash` must be a string

Comment (phthhieu) [reply client/src/Route.re]: Nice catch

**Pull request merged**